src/components/Layout/Header/banners/HeaderBanners.tsx

```diff
@@ -0,0 +1,46 @@
+import { WarningMessage } from "components/WarningMessage/WarningMessage"
```

Comment:
is that a new component? migration was done in the separate PR, right?

Reply:
HeaderBanners is new component so all banners can be lazy loaded.

Migration banner is also in this new component.